init.ps1

```diff
@@ -16,6 +15,306 @@ Demonstrates how to call this script.
 param(
 )
 
+function Start-InstallProcess
+{
+    param(
+        [string]$ExecutablePath,
+        [string[]]$ExecutableParameters
+    )
+
+    $installProcessInfo = New-Object System.Diagnostics.ProcessStartInfo
+    $installProcessInfo.FileName = $ExecutablePath
+    $installProcessInfo.RedirectStandardError = $true
+    $installProcessInfo.RedirectStandardOutput = $true
+    $installProcessInfo.UseShellExecute = $false
+    $installProcessInfo.Arguments = ($ExecutableParameters -Join ' ')
+    $installProcess = New-Object System.Diagnostics.Process
+    $installProcess.StartInfo = $installProcessInfo
+    $installProcess.Start() | Out-Null
+    $installProcess.WaitForExit()
+    $stdout = $installProcess.StandardOutput.ReadToEnd()
+    $stderr = $installProcess.StandardError.ReadToEnd()
+
+    return @{stdout = $stdout; stderr = $stderr}
+}
+
 Set-StrictMode -Version 'Latest'
 $ErrorActionPreference = 'Stop'
 $InformationPreference = 'Continue'
+$VerbosePreference = 'Continue'
+
+Write-Information -InformationAction Continue -MessageData 'Starting init.ps1 script'

```

Comment:
Instead of all the `-InformationAction Continue` statements, add this to the top of the script:

```powershell
$InformationPreference = 'Continue'
```